_inc/connect-button.js

```diff
@@ -72,7 +72,7 @@ jQuery( document ).ready( function( $ ) {
 		},
 		handleAuthorizationComplete: function() {
 			jetpackConnectButton.isRegistering = false;
-			location.reload();
+			window.location.hash = '#/plans-prompt';
```

Comment:
Should we be reloading here instead? Because at this point, the WP side nav won't be up to date if we don't reload.

Reply:
This has been addressed in https://github.com/Automattic/jetpack/pull/13348/commits/f853a0029f6d7681342665e39e5d68c041a9739b.